docs/examples/tutorial/string/return_memview.pyx

```diff
@@ -0,0 +1,7 @@
+def process_byte_data(unsigned char[:] data, bint return_all):

```

Comment:
Hmm, this is actually a semantic change. The intention was to decide inside of the function what to return, and adapt the return value/type accordingly. If you make `return_all` an argument, then this changes to "let the user decide", which is not always a good design choice.